connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -722,89 +646,68 @@ public void testRestartConnectorAndTasksBoth() throws Exception {
-        Connector connectorMock = PowerMock.createMock(SourceConnector.class);
+        Connector connectorMock = mock(SourceConnector.class);
         expectConfigValidation(connectorMock, true, connectorConfig);
 
-        // herder.stop() should stop any running connectors and tasks even if destroyConnector was not invoked
         expectStop();
 
-        statusBackingStore.put(new TaskStatus(new ConnectorTaskId(CONNECTOR_NAME, 0), AbstractStatus.State.DESTROYED, WORKER_ID, 0));
-        EasyMock.expectLastCall();
-
-        statusBackingStore.stop();
-        EasyMock.expectLastCall();
-        worker.stop();
-        EasyMock.expectLastCall();
-
-        PowerMock.replayAll();
-
         herder.putConnectorConfig(CONNECTOR_NAME, connectorConfig, false, createCallback);
         Herder.Created<ConnectorInfo> connectorInfo = createCallback.get(1000L, TimeUnit.SECONDS);
         assertEquals(createdInfo(SourceSink.SOURCE), connectorInfo.result());
 
+        // herder.stop() should stop any running connectors and tasks even if destroyConnector was not invoked
         herder.stop();
         assertTrue(noneConnectorClientConfigOverridePolicy.isClosed());
-
-        PowerMock.verifyAll();
+        verify(worker).stop();
+        verify(statusBackingStore).stop();
+        verify(statusBackingStore).put(new TaskStatus(new ConnectorTaskId(CONNECTOR_NAME, 0), AbstractStatus.State.DESTROYED, WORKER_ID, 0));
     }
 
     @Test
     public void testAccessors() throws Exception {
         Map<String, String> connConfig = connectorConfig(SourceSink.SOURCE);
         System.out.println(connConfig);
 
-        Callback<Collection<String>> listConnectorsCb = PowerMock.createMock(Callback.class);
-        Callback<ConnectorInfo> connectorInfoCb = PowerMock.createMock(Callback.class);
-        Callback<Map<String, String>> connectorConfigCb = PowerMock.createMock(Callback.class);
-        Callback<List<TaskInfo>> taskConfigsCb = PowerMock.createMock(Callback.class);
-        Callback<Map<ConnectorTaskId, Map<String, String>>> tasksConfigCb = PowerMock.createMock(Callback.class);
+        Callback<Collection<String>> listConnectorsCb = mock(Callback.class);
+        Callback<ConnectorInfo> connectorInfoCb = mock(Callback.class);
+        Callback<Map<String, String>> connectorConfigCb = mock(Callback.class);
+        Callback<List<TaskInfo>> taskConfigsCb = mock(Callback.class);
+        Callback<Map<ConnectorTaskId, Map<String, String>>> tasksConfigCb = mock(Callback.class);
 
         // Check accessors with empty worker
-        listConnectorsCb.onCompletion(null, Collections.EMPTY_SET);
-        EasyMock.expectLastCall();
-        connectorInfoCb.onCompletion(EasyMock.<NotFoundException>anyObject(), EasyMock.isNull());
-        EasyMock.expectLastCall();
-        connectorConfigCb.onCompletion(EasyMock.<NotFoundException>anyObject(), EasyMock.isNull());
-        EasyMock.expectLastCall();
```

Comment:
Aren't we missing a corresponding `doNoting().when(...)` line for this expectation?

Reply:
Committed and pushed